docs/SUMMARY.md

```diff
@@ -0,0 +1,17 @@
+# Table of contents
```

Comment:
```suggestion
# Table of Contents
```

Reply:
Maybe we need to make some of these changes to the JIRA Gitbook as well, since some of this content was copied over from there :-).